src/components/Text/Text.module.css

```diff
@@ -5,8 +5,78 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) h1, h2, h3, h4, h5, h6
```

Comment:
might be missing something -- is this comment used?

Reply:
It is not, will remove